example-crates/keychain_tracker_example_cli/src/lib.rs

```diff
@@ -13,7 +13,7 @@ use bdk_chain::{
         Descriptor, DescriptorPublicKey,
     },
     sparse_chain::{self, ChainPosition},
-    DescriptorExt, FullTxOut,
+    Append, DescriptorExt, FullTxOut,
```

Comment:
can we rewrite the examples to not use `KeychainTracker` and use `IndexedTxGraph` and `LocalChain` instead.

Reply:
Yes, separate PR?